examples/gno.land/r/demo/boards/misc.gno

```diff
@@ -78,18 +78,18 @@ func summaryOf(str string, length int) string {
 }
 
 func displayAddressMD(addr std.Address) string {
-	user := users.GetUserByAddress(addr)
+	user := users.ResolveAddress(addr)
 	if user == nil {
-		return "[" + addr.String() + "](/r/demo/users:" + addr.String() + ")"
+		return "[" + addr.String() + "](/r/gnoland/users/v1:" + addr.String() + ")"
```

Comment:
```suggestion
		return "[" + addr.String() + "](/u/" + addr.String() + ")"
```

we should not do this.

let's use `u/<handle>` now and make something simple like a http redirect before we've the final u/<handle> implementation soon (#2189).